providers/edge/src/airflow/providers/edge/models/edge_worker.py

```diff
@@ -52,6 +52,14 @@ class EdgeWorkerState(str, Enum):
     """Edge Worker was shut down."""
     UNKNOWN = "unknown"
     """No heartbeat signal from worker for some time, Edge Worker probably down."""
+    MAINTENANCE_REQUEST = "maintenance request"
+    """Maintenance mode was requested by user."""
```

Comment:
I would re-phrase, the term "user" might not be clear. Might be Admin or Ops staff mainly. But via API can also be automated.
```suggestion
    """Worker was requested to enter maintenance mode. Once worker receives this the worker will pause fetching jobs."""
```